spark-benchmarks/pom.xml

```diff
@@ -47,11 +47,11 @@
         </dependency>
         <dependency>
             <groupId>org.apache.spark</groupId>
-            <artifactId>spark-streaming-kafka-0-8_${scala.binary.version}</artifactId>
+            <artifactId>spark-streaming-kafka-0-10_${scala.binary.version}</artifactId>
         </dependency>
         <dependency>
             <groupId>org.sedis</groupId>
-            <artifactId>sedis_${scala.binary.version}</artifactId>
+            <artifactId>sedis_2.11</artifactId>
```

Comment:
Hope this does not have compatibility issues with spark 3.0 scala 2.12, if yes then i will change the code to accommodate for the same